test/ERC6538Registry.t.sol

```diff
@@ -47,24 +46,296 @@ contract RegisterKeys is ERC6538RegistryTest {
   /// forge-config: default.fuzz.runs = 1
   /// forge-config: ci.fuzz.runs = 1
   /// forge-config: lite.fuzz.runs = 1
-  function testFuzz_NeverReverts(address caller, uint256 schemeId, bytes memory stealthMetaAddress)
-    external
-  {
+  function testFuzz_AlwaysSucceeds(
+    address caller,
+    uint256 schemeId,
+    bytes memory stealthMetaAddress
+  ) external {
     vm.prank(caller);
     registry.registerKeys(schemeId, stealthMetaAddress);
   }
 }
 
 contract RegisterKeysOnBehalf_Address is ERC6538RegistryTest {
```

Comment:
Why does this have the `_Address` postfix? It prevents scopelint from generating a spec

Reply:
Removed in [a5b3e4d](https://github.com/ScopeLift/stealth-address-erc-contracts/pull/5/commits/a5b3e4db42c7230a4ae1eab0e8bca6c723c0ea1d)